src/pages/dtrader/index.tsx

```diff
@@ -2,7 +2,7 @@ import React, { useEffect, useState } from 'react'
 import styled from 'styled-components'
```

Comment:
why do we need these files?
src/pages/dbot/index_ts.swp
src/pages/dbot/.index.tsx.swp

Reply:
@prince-deriv any idea on why these binary files are changed automatically ?